plugin.py

```diff
@@ -77,3 +79,57 @@ def _cleanup_cache(cls) -> None:
         """ Clean up this plugin's cache directory. """
 
         shutil.rmtree(get_plugin_storage_dir(), ignore_errors=True)
+
+
+class LspTexlabForwardSearchCommand(LspTextCommand):
+
+    def run(self, edit):
```

Comment:
You can define the session name as a class property:

https://github.com/sublimelsp/LSP/blob/0aa408b36ca0539f14da9e8cf928639d5b079183/plugin/core/registry.py#L56-L58

That will ensure that the command is not available when there is no appropriate session.

Reply:
See: efafbaeb28169bbc3a3d3537d154e220f6af7f2f